src/lib/abacus/index.ts

```diff
@@ -64,14 +65,15 @@ import { getTransferInputs } from '@/state/inputsSelectors';
 
 import { assertNever } from '../assertNever';
 import { LocaleSeparators } from '../numbers';
+import { testFlags } from '../testFlags';
 import AbacusAnalytics from './analytics';
 import AbacusChainTransaction from './dydxChainTransactions';
 import AbacusFileSystem from './filesystem';
 import AbacusFormatter from './formatter';
 import AbacusLocalizer from './localizer';
 import AbacusLogger from './logger';
 import AbacusRest from './rest';
-import AbacusStateNotifier from './stateNotification';
+import AbacusStateNotifier, { NoOpAbacusStateNotifier } from './stateNotification';
```

Comment:
if we are blocking this.stateManager.readyToConnect is it necessary to have a NoOpAbacusStateNotifer?

Reply:
or are there defaults w/o setting readyToConnect to true that we want to prevent?

Reply:
I figured there would be some state notifications that happen even without starting the networking stuff, like setting the initial state stuff, and I wanted to prevent that.